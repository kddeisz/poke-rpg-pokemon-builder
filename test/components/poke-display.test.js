import React from "react";
import PokeDisplay from "../../src/components/poke-display";
import renderer from "react-test-renderer";

test("displays consistently", () => {
  const pokemon = { hp: 2, attack: 3, defense: 4, sAtk: 5, sDef: 6, speed: 7 };
  const component = renderer.create(<PokeDisplay poke={pokemon} level={10} />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
