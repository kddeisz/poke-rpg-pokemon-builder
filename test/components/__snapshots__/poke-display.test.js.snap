// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays consistently 1`] = `
<div
  className="well poke-display clearfix"
>
  <div
    className="col-xs-6"
  >
    <ul
      className="list-unstyled text-right strong"
    >
      <li>
        HP
      </li>
      <li>
        Attack
      </li>
      <li>
        Defense
      </li>
      <li>
        Special Attack
      </li>
      <li>
        Special Defense
      </li>
      <li>
        Speed
      </li>
    </ul>
  </div>
  <div
    className="col-xs-6"
  >
    <ul
      className="list-unstyled"
    >
      <li>
        26
      </li>
      <li>
        3
      </li>
      <li>
        4
      </li>
      <li>
        5
      </li>
      <li>
        6
      </li>
      <li>
        7
      </li>
    </ul>
  </div>
</div>
`;
